tutorials/cp-cap-java-hana-db/cp-cap-java-hana-db.md

```diff
@@ -65,7 +65,7 @@ First you need to create and initialize an SAP HANA database schema in SAP BTP.
 
 You first need to provision your SAP HANA Cloud instance, which is a prerequisite to later on create a SAP HANA HDI Container to deploy your database artifacts to.
 
-1. Follow the tutorial [Provision an Instance of SAP HANA Cloud](hana-cloud-mission-trial-2). Use `bookstore-db` as the name of your database and make sure to allow access to your SAP HANA Cloud from all IPs.
+1. Follow the tutorial [Provision an Instance of SAP HANA Cloud](hana-cloud-mission-trial-2). Use `bookstore-db` as the name of your database. Make sure to allow access to your SAP HANA Cloud from all IPs and that instance of the SAP HANA you have created is mapped to your subaccount and space where you working with this tutorial.
```

Comment:
Have you tried that in a trial account? I guess this is not needed in trial accounts, which is the assumption for this tutorial.